apps/ledger-live-mobile/src/screens/Platform/LiveApp.tsx

```diff
@@ -21,7 +21,7 @@ type Props = StackNavigatorProps<BaseNavigatorStackParamList, ScreenName.Platfor
 
 export function LiveApp({ route }: Props) {
   const { theme } = useTheme();
-  const { platform: appId, customDappURL, ...params } = route.params || {};
+  const { platform: appId, ...params } = route.params || {};
```

Comment:
Careful adding stuff back in the `params` rest value, it's only meant for the `inputs` passed to the webview, please revert the change and add the `customDappUrl` here too